streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBSegmentedBytesStore.java

```diff
@@ -55,6 +55,18 @@
     }
 
     @Override
+    public KeyValueIterator<Bytes, byte[]> fetch(final Bytes keyFrom, Bytes keyTo, final long from, final long to) {
```

Comment:
underlying byte ranges differ significantly for single keys vs key range, so I kept both methods